README.md

```diff
@@ -66,7 +67,7 @@ A valid (read: that follows this interface) IPFS core implementation, must expos
 
 `callback` must follow `function (err, node) {}` signature, where `err` is an error if the operation was not successful and `node` is a MerkleDAG node of the type [DAGNode][]
 
-If no `callback` is passed, a promise is returned.
+Returns a [promise][].
```

Comment:
Alas! I thought so too, but tried this with `promisify-es6` -- [it returns a promise regardless](https://gist.github.com/noffle/aac170fc16890c77809f0ba14d9fa235).